pywinauto/uia_element_info.py

```diff
@@ -95,15 +90,17 @@ def __init__(self, handle_or_elem = None):
                     "with integer or IUIAutomationElement instance only!")
         else:
             self._element = IUIA().root
-
-        self.set_cache_strategy(False)
+ 
+        self.set_cache_strategy(cache_enable)
 
     def _get_current_class_name(self):
         """Return an actual class name of the element"""
         return self._element.CurrentClassName
     
     def _get_cached_class_name(self):
         """Return a cached class name of the element"""
+        if self._cached_class_name == False:
```

Comment:
This comparison produces Landscape warning. Do you mean that `self._cached_class_name == "" or None` is not the case?